### ISSUE TITLE
A11y_Visual Studio Code Services_Docs_Left Right Navigation_UseofColor :  The users will not be able to differentiate between the normal text and link.

### ISSUE THREAD

**Issue by jadhavprathmesh01** [microsoft/vscode-docs#5105]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #UseofColor;#WCAG1.4.1; #FTP; #ChromiumEdge

**Environment Details:**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps:**
1. Launch the application Visual Studio Code Services
2. Tab Navigate to "Docs" link.
3. Left and right navigation will appear.
4. Tab navigate to the right navigation links(eg. VS code in Action, First step etc.) and observe that the keyboard users can distinguish between the normal text and link.

**Actual:**
The keyboard users will not be able to differentiate between a link (eg. VS code in Action, First step etc.) and normal text, there is no underline provided or any use of color to differentiate between links (eg. VS code in Action, First step etc.) and normal text.

**Expected Result:**
The users should be able to distinguish between the normal text and links i.e. links should be provided with a underline or use of color, so that users will be able to understand between a link and normal text.

**User Impact:**
The links should be descriptive so that users will be able to distinguish between the normal text and link.

**Attachment:**
![A11y_Visual Studio Code Services_Docs_Left Right Navigation_UseofColor](https://user-images.githubusercontent.com/87858881/149919919-58c6778e-5797-4097-95da-d6780d376229.gif)

**Comment by sspatil123** [comment]
![image](https://user-images.githubusercontent.com/85863999/217257641-12244be8-7f9e-4b14-8378-492fe5281238.png)
GithubTags:#Closed;Version 1.75
